option optimize_for = LITE_RUNTIME;

message TowerLadderEliteRequest
{
 required uint32 id = 1;
}
